Lib/fontTools/varLib/mutator.py

```diff
@@ -123,6 +136,21 @@ def instantiateVariableFont(varfont, location, inplace=False):
 			if n.nameID not in exclude
 		]
 
+	if "wght" in location and "OS/2" in varfont:
+		varfont["OS/2"].usWeightClass = otRound(
+			max(1, min(location["wght"], 1000))
+		)
+	if "wdth" in location:
+		wdth = location["wdth"]
+		for percent, widthClass in sorted(OS2_WIDTH_CLASS_VALUES.items()):
+			if wdth <= percent:
```

Comment:
Not sure if this is correct, or if we should round in a more clever way, testing in the middle of those predefined percent intervals...